package.json

```diff
@@ -7,8 +7,8 @@
     "url": "https://github.com/AdobeDocs/commerce-marketplace"
   },
   "dependencies": {
-    "@adobe/gatsby-theme-aio": "4.6.6",
-    "gatsby": "4.22.0",
+    "@adobe/gatsby-theme-aio": "4.8.3",
+    "gatsby": "4.25.5",
```

Comment:
Apparently, the aio-theme doesn't support gatsby 4.25.5. See https://github.com/AdobeDocs/commerce-webapi/pull/101#discussion_r1130160823.

Reply:
Confirmed. I found it in the aio-theme too: `make sure it is using gatsby version 4.22.0.` (https://github.com/adobe/aio-theme#developing-with-adobe-theme-io). Should be more visible.
Fixed.